detections/endpoint/office_application_spawned_cmd_shell.yml

```diff
@@ -0,0 +1,50 @@
+name: Office Application Spawned CMD Shell
+id: c1ad8f62-9c65-11eb-9d59-acde48001122
+version: 1
+date: '2021-04-13'
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: this search is to detect a potential malicious office macro document
+  that spawn command shell to execute malicious batch, powershell, vb/js script. this
+  technique is common on malicious ms office macro document used by threat actor to
+  load payloads, download payloads or other malware.
+search: '| tstats `security_content_summariesonly` count values(Processes.process)
```

Comment:
@t-contreras :we have a similar search that looks for this behavior based on a lookup file: https://github.com/splunk/security_content/blob/develop/detections/endpoint/detect_prohibited_applications_spawning_cmd_exe.yml

this might be redundant

Reply:
And we would need a story file for : Spearphishing Attachments

Reply:
thanks again Bhavin, I removed already that file and will rename the analytic story back to Phishing Payload. ;)